blogposts/2022/release-3.39

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Sourcegraph 3.39 release'
+publishDate: 2022-04-21T10:00-07:00
+description: 'Sourcegraph 3.39 introduces...'
+tags: [blog, release]
+slug: 'release/3.39'
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-39-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-39-release.png
+changelogItems:
+
+---
+
+Sourcegraph 3.39 is now available! In this release:
+
+## Create living documentation with Notebooks, now Generally Available
+
+## Code Insights is no longer backed by Timescale
+
+Code Insights can now be used with managed database solutions that do not support [Timescale](https://www.timescale.com/), like Amazon RDS. It now uses native Postgres.
+
+This migration has been applied over 3.37 and 3.38, and should be seamless if you are running infrastructure as described by the [official Sourcegraph deployment repositories](https://sourcegraph.com/search?q=context:%40sourcegraph/all+repo:%5Egithub%5C.com/sourcegraph/deploy-sourcegraph+select:repo&patternType=literal). If you have a customized deployment, you may need to incorporate some changes into your configuration ([more details and examples here](https://github.com/sourcegraph/sourcegraph/issues/32271#issuecomment-1086328666)).
+
+Any managed database solution that supports the [official Sourcegraph Postgres version requirements](https://docs.sourcegraph.com/admin/postgres#version-requirements) is now suitable for Code Insights. Migration from a self-hosted database is not performed automatically, and will need to be performed by a Sourcegraph admin at your own discretion.
+
+## Improved search-based local navigation for Java
+
+We've added tree-sitter to our search-based recipe to make our local navigation for Java faster and more precise.
+
+<video class="blog-image" title="Java code Intelligence" alt="An example of using Java Code Intelligence within the sourcegraph/sourcegraph repo." loop autoplay muted playsinline>
+  <source src="https://user-images.githubusercontent.com/1657213/163855878-4355bcb0-5407-4307-b3f2-4581e1ab2fe2.mp4">
+</video>
+
+## Search your Go dependenices
+
+[Dependencies search](https://docs.sourcegraph.com/code_search/how-to/dependencies_search) allows you to quickly search through both your code and dependencies for faster incident and bug resolution. Dependencies search now supports Go dependencies in addition to NPM.
```

Comment:
@eugeniaft Could you please take a look at this revision? Is it fair to say "...for faster incident and bug resolution" ?

Reply:
Maybe we can say: "... for faster incident and vulnerabilities resolution." I would also change the first part: "... allows you to quickly search through your repo dependencies..."